### DISCUSSION TITLE
Question:  Two basement on the both sides of a fault

### DISCUSSION THREAD

**Discussion by Changwan-planet** [gempy-project/gempy#814] [General]
Hello there, 

Thank you for opening your awesome program. I am actively using Gempy to answer my research question related paleoseismology.
My target is usually related to soft sediment within 5 m depth, and I got many trench sections.
Based on my knowledge from trench info, I would like to make a 3d geological model in my research site using your program.

I am a beginner of the Gempy.

Your program always make a "basement layer" automatically.  
But, there is a fault between two lithological basements in my site.

How can I make two basement layers automatically in my model?

I attached my Points and Orientation data. 
I am testing the factors of orientation. So, the model looks strange.

[Miho_Points_6_nofault.csv](https://github.com/cgre-aachen/gempy/files/12156671/Miho_Points_6_nofault.csv)
[Miho_Orientations2_nofault.csv](https://github.com/cgre-aachen/gempy/files/12156683/Miho_Orientations2_nofault.csv)

I hope to learn a lot from this discussion.

Thanks

**Comment by AlexanderJuestel** [comment]
Hello @Changwan-planet, 

could you maybe provide a sketch of how the situation looks like so we can have a better imagination of your case.
Cheers
Alex

**Comment by Changwan-planet** [comment]
Hello @AlexanderJuestel, 

Really thank you for your reply.

First of all, I attatched new *.csv files for using gempy.
Also, I uploaded the sketch of my data in the *.csv files.

Info. for a sketch
Miho : name of my research site.
T :trench
Numbers : Date of excavation e.g.) 221130 means November 30, 2022.
fl : fluvial
w : west
e : east

Miho_Points.csv includes the points of the surfaces above the Basement_1 and Basement_2. I call the surface above the Basement_1 as T_221123_f1_w_1 and T_221130_f1_w_1. The reason I divide the suface into two names is that they come from different trench sections. Also I name the surface above the Basement_2 as T_221130_f1_e_1. I did not include a fault in my data yet.

At this momment, my first goal is to model the suface above the basement (Basement_1 and Basement_2) based on trench sections. Excecpt the uploaded data, I have more and more information from trench sections. I imagine I will be able to make a geological model like "Greenstone" example in your real case studies. But that has only one basement.  

Please help me out... If you need more information for a better understanding of my project, Please leave some comments.

[Miho_Orientations.csv](https://github.com/cgre-aachen/gempy/files/12236985/Miho_Orientations.csv)
[Miho_Points.csv](https://github.com/cgre-aachen/gempy/files/12236986/Miho_Points.csv)
[Miho_Sketch_1.pdf](https://github.com/cgre-aachen/gempy/files/12237085/Miho_Sketch_1.pdf)

**Comment by AlexanderJuestel** [comment]
Just posting your sketch here. Would it be possible to model the "Base Sediments" which would be equal to the "Top Basement" and just define one basement that consists of both your basement lithologies? Or is it necessary to distinguish them?

![image](https://github.com/cgre-aachen/gempy/assets/45469915/39892df4-2260-487b-9dc9-997451113752)

**Reply by Changwan-planet** [reply]
OMG. Really quick reply!
@AlexanderJuestel  Thank you so much. 

I keep failing to upload the image. I got this message with the file types, *.png, *.jpg, and *. gif: Somthing went really wrong, we can't process that file. Then, I made a pdf. Please understand this issue.

Yeah, I thought if I trick gempy like your suggestion as well.
My answer is that we have to distinguish them because they have distinct lithology. 

We had 3D trenches in the research site. For a better understanding, I attached the birdview of our work.
[Map_view2.pdf](https://github.com/cgre-aachen/gempy/files/12237660/Map_view2.pdf)

When you look at the Map_view2 file, I only uploaded the information from 1 and 3 trenches in the previous post. I have more and more data. So I think we can make a geological model in this local area.

**Reply by AlexanderJuestel** [reply]
You can just copy and paste a screenshot, for instance, that works very smoothly. 

One idea that comes to my mind right now would be to build two models, one with your yellow basement and one with your pink basement and intersect these afterward to get a final model.

**Reply by Changwan-planet** [reply]
@AlexanderJuestel 
Many thanks for you. I will reopen a new discussion with models that you suggested.
